ods-tutorial/source/dashboard_modules/how_to_create_tabs.rst

```diff
@@ -0,0 +1,546 @@
+
+Let's take a look at the syntax to understand how to use the ``ng-if`` directive.
+
+.. code-block:: javascript
+
+ ng-if="myvariable == 'myvalue'"
+
+The structure of the ``ng-if`` directive is the same as for ``ng-click``. The name of the directive is followed by ``=`` and the expression that will be evaluated by the ``ng-if`` directive is written between ``"``.
+
+But the syntax differs a little bit. In the expression,
+
+- The variable is the same we created in step 8.
+- Here is where the syntax differs: this time, the variable and value must be separated by ``==``. It indicates that we express a condition.
+- The value (written between ``'``) is the same we created in step 8.
+
+Basically, the ``ng-if`` syntax could be read as such: "If ``myvariable`` equals ``myvalue``, [display the content of ``myvalue``]".
+
+.. rst-class:: block-step
+
+    11 + Add the ``ng-if`` variable in the HTML ``<div>`` tag of each tab content. Don't forget to put all 3 values to match the 3 tab contents. At this point, we have indicated AngularJS which tab content to display depending on which tab title is selected.
+
+.. code-block:: html
+
+ <div class="ods-box">
+   <div ng-if="tabselector == 'firsttab'">
+     Content of first tab
+   </div>
+   <div ng-if="tabselector == 'secondtab'">
+     Content of second tab
+   </div>
+   <div ng-if="tabselector == 'thirdtab'">
+     Content of third tab
+   </div>
+ </div>
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    12 + It's time to tackle ``ng-init``! This directive is used to initialize a variable: it basically means that when a web page is loaded, the code behind that page is going to be read from the first to the last line. ``ng-init`` will indicate that from there, AngularJS will be used, and which variable will be used. In our case, ``ng-init`` will also be used to indicate which tab content should be displayed by default.
+
+Let's take a look at the syntax to understand how to use the ``ng-init`` directive.
+
+.. code-block:: javascript
+
+ ng-init="myvariable = 'myvalue'"
+
+The syntax of the ``ng-init`` directive is exactly the same as for ``ng-click`` (see step 8).
+
+In the expression:
+
+- The variable is the same we created in step 8 and have been using since.
+- The value (written between ``'``) is the identifier of the tab which content will be display by default.
+
+.. rst-class:: block-step
+
+    13 + Add the ``ng-init`` variable in an HTML tag that wraps the whole code of the tabs. For this tutorial, we are going to put ``ng-init`` in the default ``<div class="ods-box">`` tag that is by default in all OpenDataSoft page (right below ``<div class="container">`` - both tags being the default frame of any OpenDataSoft page). At this point, our tabs are fully functionable!
```

Comment:
The default template is like this 
```
<div class="container-fluid">
    <div class="ods-box" ></div>
</div>
```
it's a container fluid, not a container, and there is an ods-box already.

it's very close but it can be confusing for non-techs ? what do you think ?

As you ask to add a surrounding div  with ods-box in previous steps, there will be several ods-box... 

The full code exemple right after is ok for people to understand, but I just wanted to comment :)